stable/artifactory-ha/OWNERS

```diff
@@ -0,0 +1,14 @@
+approvers:
+- unguiculus
+- prydonius
+- mattfarina
```

Comment:
Can you remove the repo maintainers from this and the reviewers list. Since we are on the repo we do not need to be listed here.